text/0000-better-env-handling.md

```diff
@@ -0,0 +1,87 @@
+- Start Date: 2020-04-23
+- Relevant Team(s): Ember.js, Learning
+- RFC PR: [#619](https://github.com/emberjs/rfcs/pull/619)
+- Tracking: (leave this empty)
+
+# Better environment handling
+
+## Summary
+
+This RFC proposes to add a `TESTING` export to `@glimmer/env`, to allow to more easily work with common environment questions.
+Additionally, it proposes to add the `@glimmer/env` module to the [API docs](https://api.emberjs.com/), like `@glimmer/tracking`.
+
+## Motivation
+
+Currently, there is no _one_ way to identify if the application is running in test mode.
+
+Some reasons why you would want to check for that in your code include:
+
+- Disabling polling code in tests
+- Preventing test-breaking code like window.reload()
+- Preventing log output in tests
+
+There are three main ways to solve this, as of now:
+
+1. `Ember.testing`
+
+This is not really recommended anymore. There is no module export for this (you need to use the `ember` import for it).
```

Comment:
We should also mention that it cannot be used in module scope at all ([see `no-ember-testing-in-module-scope` eslint-plugin-ember](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-ember-testing-in-module-scope.md) rule).

Reply:
I have added this.